modules/virt-assigning-pci-device-virtual-machine.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-assigning-pci-device-virtual-machine_{context}"]
+= Assigning a PCI device to a virtual machine
+
+When a PCI device is available in a cluster, you can assign it to a virtual machine and enable PCI passthrough.
+
+.Procedure
+. Assign the PCI device to a virtual machine as a host device.
```

Comment:
Because this procedure is just one step, it should be a bullet point (`*`) instead of an ordered list

Reply:
Added the bullet.